Dockerfile

```diff
@@ -31,5 +31,6 @@ RUN wget -q "https://storage.googleapis.com/kubernetes-release/release/v${KUBECT
  && chmod +x "/usr/bin/kubectl" \
  && chmod +x "/usr/bin/kd"
 
+COPY util.sh /util.sh
```

Comment:
I think we should give this a more descriptive name. `get-npm-package-version.sh`? :-/